main.py

```diff
@@ -2,6 +2,43 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    [
+        Genre.objects.create(name=genre) for genre in
+        [
+            "Western",
+            "Action",
+            "Dramma",
+        ]
+    ]
+
+    [
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name
+        )
+        for first_name, last_name in
+        [
+            ("George", "Clooney"),
+            ("Keanu", "Reeves"),
+            ("Scarlett", "Keegan"),
+            ("Will", "Smith"),
+            ("Jaden", "Smith"),
+            ("Scarlett", "Johansson"),
+        ]
+    ]
```

Comment:
Do not use list comprehensions if you do not intend to store the result in a variable.